pontoon/uxactionlog/migrations/0002_update_locale_code.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 4.2.11 on 2024-07-24 18:49
+
+from django.db import migrations
+
+
+def update_locale_code(apps, schema_editor):
+    UXActionLog = apps.get_model("uxactionlog", "UXActionLog")
+
+    for log in UXActionLog.objects.filter(action_type="LLM Dropdown Select"):
+        data = log.data
+        if "targetLanguage" in data:
+            data["localeCode"] = data.pop("targetLanguage")
```

Comment:
Forgot to point that out earlier (sorry!): `targetLanguage` stores the value of a Locale.name. So we need to convert it to Locale.code when storing it to `localeCode`.